xAPI.md

```diff
@@ -804,9 +830,9 @@ _Semantics_
 
 The IRI represented by the Verb id identifies the particular semantics of a word, not the word itself. 
 
-For example, the English word "fired" could mean different things depending on context, such as "fired a 
-weapon", "fired a kiln", or "fired an employee". In this case, an IRI MUST identify one of these specific 
-meanings, not the word "fired". 
+For example, the English word "fired" could mean different things depending on context, such as 
+"fired(a weapon)", "fired(a kiln)", or "fired(an employee)". In this case, an IRI identifies one of 
+these specific meanings. 
```

Comment:
Need some slight fixes with Display vs display.  If we use the word property, please use "display", otherwise Display is okay.  I'm not sure we need to call them Verb Displays, either.  I think you do a good job in the one instance you are talking about the canonical value of display vs. the display in the Verb.